CHANGELOG.md

```diff
@@ -11,6 +11,11 @@
 * [FEATURE] Alertmanager: Added rate-limits to notifiers. Rate limits used by all integrations can be configured using `-alertmanager.notification-rate-limit`, while per-integration rate limits can be specified via `-alertmanager.notification-rate-limit-per-integration` parameter. Both shared and per-integration limits can be overwritten using overrides mechanism. These limits are applied on individual (per-tenant) alertmanagers. Rate-limited notifications are failed notifications. It is possible to monitor rate-limited notifications via new `cortex_alertmanager_notification_rate_limited_total` metric. #4135 #4163
 * [FEATURE] Alertmanager: Added `-alertmanager.max-config-size-bytes` limit to control size of configuration files that Cortex users can upload to Alertmanager via API. This limit is configurable per-tenant. #4201
 * [FEATURE] Added flag `-debug.block-profile-rate` to enable goroutine blocking events profiling. #4217
+* [FEATURE] Alertmanager: The experimental sharding feature is now considered complete. Detailed information about the configuration options can be found (here for alertmanager)[https://cortexmetrics.io/docs/configuration/configuration-file/#alertmanager_config] and (here for the alertmanager storage)[https://cortexmetrics.io/docs/configuration/configuration-file/#alertmanager_storage_config]. To use the feature:  #4222
```

Comment:
> To use the feature:  #4222

Could you add the PR numbers of all other PRs that build up the feature, please?